sae_lens/training/sae_trainer.py

```diff
@@ -287,6 +287,8 @@ def _build_train_step_log_dict(
             "losses/mse_loss": mse_loss,
             "losses/l1_loss": l1_loss
             / self.current_l1_coefficient,  # normalize by l1 coefficient
+            "losses/sfn_sparsity_loss": output.sfn_sparsity_loss,
```

Comment:
Not convinced we should have a different loss for SFN since for vanilla SAE we use the l1 loss. I think we could change what we call the l1 loss to sparsity loss or something though.